package.json

```diff
@@ -12,17 +12,17 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^2.6.0",
-    "@typescript-eslint/parser": "^2.6.0",
-    "eslint": "^7.32.0",
+    "@typescript-eslint/eslint-plugin": "^5.4.0",
+    "@typescript-eslint/parser": "^5.4.0",
+    "eslint": "^8.2.0",
     "ts-node": "^10.4.0",
     "typedoc": "^0.22.9",
-    "typescript": "^4.3.5"
+    "typescript": "^4.4.4"
   },
   "scripts": {
     "build": "npm run lint && tsc",
     "doc": "typedoc --out docs ./src",
-    "lint": "eslint src/**/*.ts"
+    "lint": "eslint src/*.ts src/**/*.ts"
```

Comment:
I also thought just `src/**/*.ts` is enough, but when I run `npm run lint` it will only lint files in sub folders, not the files in `src/`, so I added `src/*.ts`.

Reply:
Interesting. I've also used in other projects simply `eslint src`, without any globbing, which works for subdirectories as well.

Reply:
The `eslint src` works well, updated, thanks!